src/modules/commander/commander_params.c

```diff
@@ -749,6 +749,30 @@ PARAM_DEFINE_FLOAT(COM_POS_FS_EPV, 10);
  */
 PARAM_DEFINE_FLOAT(COM_VEL_FS_EVH, 1);
 
+/**
+ * FailureDetector Max Roll
+ *
+ * Maximum roll angle before FailureDetector triggers the attitude_failure flag
+ * Does not affect the behavior of the vehicle for now; only for logging
+ * @min 0
+ * @max 180
+ * @unit degrees
+ * @group Commander
+ */
+PARAM_DEFINE_INT32(COM_FAIL_R, 60);
```

Comment:
If you foresee the list of FailureDetector parameters growing I'd break it off into a new group (and param file) now.